Makefile

```diff
@@ -69,7 +86,7 @@ debug: build-debug
 # Remove compiled binaries, output files, and temp files
 clean:
 	@ find ./bin/ ! -name '.gitignore' -type f -exec rm -f {} +
-	@ find tests/workdir/output ! -name '.gitignore' -type f -exec rm -f {} +
-	@ find tests/workdir/temp ! -name '.gitignore' -type f -exec rm -f {} +
-	@ find workdir/output ! -name '.gitignore' -type f -exec rm -f {} +
-	@ find workdir/temp ! -name '.gitignore' -type f -exec rm -f {} +
+	@ rm -rf workdir/temp
+	@ rm -rf workdir/output
+	@ rm -rf tests/workdir/temp
+	@ rm -rf tests/workdir/output
```

Comment:
FileManager manages creating the directories if they don't exist, and we've removed the `.gitignores` in a previous commit for the same reason. Simplified this a bit.

Reply:
LGTM